views/components/Matrix.d3.js

```diff
@@ -300,155 +521,239 @@ export default class Matrix extends AbstractGraph {
+    if (xAxisForDatesData.length > 0) {
+      this.xAxisLinearBottom = d3
+        .axisBottom(this.xScaleLinearBottom)
         .ticks(xAxisForDatesData.length - 1)
         .tickValues(xAxisValuesBottom)
-        .tickFormat(d3.format("d"))
+        .tickFormat(d3.format('d'))
 
-      this.xAxisBottom = this.svg.append("g")
-        .attr("class", "xAxisLinearBottom")
-        .attr("transform", (d, i) => { return "translate(0," + xAxisBottomYCoordinate + ")" })
+      this.xAxisBottom = this.svg
+        .append('g')
+        .attr('class', 'xAxisLinearBottom')
+        .attr('transform', (d, i) => {
+          return 'translate(0,' + xAxisBottomYCoordinate + ')'
+        })
         .call(this.xAxisLinearBottom)
-      this.xAxisBottom.selectAll('text')
-        .attr("font-size", (d, i) => {
+      this.xAxisBottom
+        .selectAll('text')
+        .attr('font-size', (d, i) => {
           if (d.length <= 3) {
             return this.cardSize / 2
           }
-          return (this.cardSize / (d.length - 1.5))
+          return this.cardSize / (d.length - 1.5)
         })
-        .style("fill", "#545454")
+        .style('fill', '#545454')
 
-      this.xAxisBottom.selectAll('path')
-        .attr('stroke-opacity', 0)
-      this.xAxisBottom.selectAll('line')
-        .attr('stroke-opacity', 0)
+      this.xAxisBottom.selectAll('path').attr('stroke-opacity', 0)
+      this.xAxisBottom.selectAll('line').attr('stroke-opacity', 0)
     }
-    this.xAxisTop = this.svg.append("g")
-      .attr("class", "xAxisLinearTop")
+    this.xAxisTop = this.svg
+      .append('g')
+      .attr('class', 'xAxisLinearTop')
       .call(this.xAxisLinearTop)
 
-    this.xAxisTop.selectAll("text")
-      .data(xAxisForDatesData.filter((d, i) => { return (d.day >= startDayForFilter && d.day <= lastDayForFilter) }))
-      .text((d) => { return d.marker })
-      .attr("font-size", this.cardSize / 2)
-      .attr("class", (d, i) => { return (d.marker == 'N/A') ? "EmptyDates" : "DatesText" })
-      .style("fill", (d, i) => { return (d.marker == 'N/A') ? "#ffffff" : "#545454" })
-      .on("click", (d, i) => {
+    this.xAxisTop
+      .selectAll('text')
+      .data(
+        xAxisForDatesData.filter((d, i) => {
+          return d.day >= startDayForFilter && d.day <= lastDayForFilter
+        })
+      )
+      .text((d) => {
+        return d.marker
+      })
+      .attr('font-size', this.cardSize / 2)
+      .attr('class', (d, i) => {
+        return d.marker == 'N/A' ? 'EmptyDates' : 'DatesText'
+      })
+      .style('fill', (d, i) => {
+        return d.marker == 'N/A' ? '#ffffff' : '#545454'
+      })
+      .on('click', (d, i) => {
         let day = d.day
-        let link = '/deepdive/' + this.props.study + '/' + this.props.subject + '/' + day
+        let link =
+          '/deepdive/' + this.props.study + '/' + this.props.subject + '/' + day
         window.open(`${basePath}${link}`, '_blank')
       })
```

Comment:
I think we can remove this. The page is being unused https://gnardog.slack.com/archives/C03KGHSSTNC/p1688340722944729?thread_ts=1687804987.105369&cid=C03KGHSSTNC
```suggestion

```